low-level/requirements.txt

```diff
@@ -8,4 +8,3 @@ zope.event==4.4
 zope.interface==4.7.1
 pyinotify==0.9.4
 python-consul==1.1.0
-cryptography==2.8
```

Comment:
Don't we need this anymore?

Reply:
We are using cortx-utils for decrypting passwords now.